uwblueprint/headsupguys
# Feature/connect tool endpoints

## Notion ticket link

[Connect 'tool' GET route to dashboard](https://www.notion.so/uwblueprintexecs/Connect-tool-GET-route-to-dashboard-dccfbe61584042a090ca927b3165d6d2)



## Implementation description
* Used axios to call the api endpoint for getting all the tools
* Needed to add `createdBy` key to the database model/schema and seeding script for tools because it wasn't initially there



## Steps to test
1. Re-seed your database with `yarn seed` to get the createdBy field for tools
2. Check `admin/dashboard/tools` to see the tools loaded from the database 



## What should reviewers focus on?
* Seeing the tools on `admin/dashboard/tools` 
* The reseeding/editing of the tools schema for the `createdBy` string of arrays that was added


## Checklist
